Client/src/Components/Inputs/Autocomplete/index.css

```diff
@@ -0,0 +1,9 @@
+.auto-complete-field div {
+  width: var(--env-var-width-2);
+  height: 34px;
+  border-radius: 4px;
+}
+
+.auto-complete-field div div .MuiOutlinedInput-root {
+  padding-left: 0;
+}
```

Comment:
I believe we should be using the theme for most if not all of these values?

Reply:
@ajhollid 

This is the smallest I could write this code. Neither of these classes could be targeted from the `jsx` file.
Wherever I added the `height` and removed it from here, the effect was gone and the height of the autocomplete got bigger.

Like this:

![Screenshot 2024-09-03 122610](https://github.com/user-attachments/assets/dba1f482-96d0-4bfd-a8b2-dc9b443e700b)

And about the padding, this classes is a nested class inside the `TextField` component and can not be targeted by the properties provides by the component itself, as below:

![Screenshot 2024-09-03 122936](https://github.com/user-attachments/assets/952202f8-52ac-43cf-9fe0-71376ecb55cc)
![Screenshot 2024-09-03 122907](https://github.com/user-attachments/assets/afd0102d-e2cc-4f0e-960c-7c0bd75ccb2c)

Do you have any ways for fixing this?

Reply:
Have you discussed the issue with the rest of the FE team?  They would be the best people to ask this question to :+1:   I'm sure there's a way to deal with it.

If the rest of the FE team agrees that there's no way to do this other than using the CSS above then we'll have to live with it.